src/bilder/images/concourse/deploy.py

```diff
@@ -96,11 +97,14 @@
             "{{ .Data.data.github_client_secret }}"
             "{{ end }}"
         ),
+        vault_url="http://localhost:8200",
+        vault_client_token="this-token-gets-overridden-by-the-vault-agent",
+        vault_path_prefix="/secret-concourse",
     ),
     CONCOURSE_WORKER_NODE_TYPE: partial(
         ConcourseWorkerConfig,
-        ephemeral=True,
-        container_runtime="containerd",
+        container_runtime="guardian",
```

Comment:
Why guardian?

Reply:
Because it is the current default and when trying to use containerd it led to the worker failing to start up. Containerd is the new preferred runtime, but we'll have to do some additional debugging to make it work.